caikit/runtime/utils/import_util.py

```diff
@@ -80,14 +81,15 @@ def get_data_model(config: ConfigParser = None) -> UnifiedDataModel:
         running server instance.
 
     Args:
-        config(ConfigParser): Config parser instance
+        config(aconfig.Config): caikit configuration
 
     Returns:
         (module): Handle to the module after dynamic wild import
     """
     if not config:
-        config = ConfigParser.get_instance()
-    lib_names = clean_lib_names(config.caikit_library)
+        config = get_config()
+    # EJ: whyyyy
```

Comment:
NIT: Let's get rid of this comment. As to `whyyyy`, this is so that library names can include things like the version specifier. It was originally done so that a given `Dockerfile` only needed to specify its input libraries once and could include versions. Those would then be stripped of here so that it was just the lib name for importing. That's an admittedly brittle reason!

Reply:
I actually accidentally left this in here to keep track of the config usage and debugging the tests that used `SimpleNamespace` to make a mock config, and with the switch to nested keys, the nested layer of config was only accessible via keys and not by attribute. This was fixed by updating all the tests to `temp_config`. I'll try to purge this.